lib/matplotlib/tests/test_preprocess_data.py

```diff
@@ -73,10 +74,18 @@ def test_function_call_without_data(func):
             "x: ['x'], y: ['y'], ls: x, w: xyz, label: text")
 
 
+@pytest.mark.parametrize('func', all_funcs, ids=all_func_ids)
+def test_function_call_with_dict_input(func):
+    """Tests with dict input, unpacking via preprocess_pipeline"""
+    data = {'a': 1, 'b': 2}
+    assert(func(None, data.keys(), data.values()) ==
+            "x: ['a', 'b'], y: [1, 2], ls: x, w: xyz, label: None")
+
+
 @pytest.mark.parametrize('func', all_funcs, ids=all_func_ids)
 def test_function_call_with_dict_data(func):
     """Test with dict data -> label comes from the value of 'x' parameter."""
-    data = {"a": [1, 2], "b": [8, 9], "w": "NOT"}
+    data = {'a': [1, 2], 'b': [8, 9], "w": "NOT"}
```

Comment:
What is the motivation to change the quotes? And why not for "w"?

Reply:
Honestly I think this was an accident while I was trying to fix some flake error. Can undo.